pds_doi_core/util/doi_validator.py

```diff
@@ -46,6 +48,25 @@ def get_database_name(self):
     def __lidvid(columns, row):
         return f"{row[columns.index('lid')]}::{row[columns.index('vid')]}"
 
+    def _check_field_site_url(self, doi: Doi):
+        """ If the site_url field exist in doi object, check to see if it is online.  If the site is not online, an exception will be thrown.
+        """
+
+        logger.debug(f"doi {doi}")
+        logger.info(f"doi.site_url {doi.site_url}")
+
+        if doi.site_url:
+            try:
+                request = requests.get(doi.site_url,timeout=5)
```

Comment:
request is maybe not the best name for this variable since this is the response.

Reply:
You are right.  Will change to a better name.

Reply:
ok cool